configure.ac

```diff
@@ -588,7 +596,7 @@ AS_IF([test x"$with_cuda" != x"no"],
 				 [cuda_runtime.h],
 				 [cudart],
 				 [cudaMemcpy],
-				 [-lcuda],
+				 [-lcuda -lnvidia-ml],
```

Comment:
I think this change is already in libfabric main so shouldn't be present in your PR. I didn't see this in the "Files changed" so there is something broken in your commit cherry-pick

Reply:
@shijin-aws this modification was added to make the file consistent with main. The reference `-lnvidia-ml` is supposed to be there and the PR had it removed; I went back and reapplied the cleanup process you provided to hopefully resolve this issue.

Thank you again for your patience working with me on this PR.